citrination_client/search/client.py

```diff
@@ -48,7 +48,7 @@ def _validate_search_query(self, pif_system_returning_query):
         if size < 0:
             raise CitrinationClientError("Size cannot be negative. Please enter a value greater than or equal to zero")
         if start_index + size > MAX_QUERY_DEPTH:
-            raise CitrinationClientError("Citrination does not support pagination past the {}th result. Please change from_index and size arguments to be within this limit".format(MAX_QUERY_DEPTH))
+            raise CitrinationClientError("Citrination does not support pagination past the {}th result. Please reduce either the from_index and/or size such that their sum is below {}".format(MAX_QUERY_DEPTH, MAX_QUERY_DEPTH))
```

Comment:
I believe you can just use MAX_QUERY_DEPTH once if you write your format string like:
`"blah blah blah {0} blah blah blah {0}".format(variable)`

Reply:
And, if we didn't have to support old python, you could do: `f"blah blah blah {variable} blah blah blah {variable}"`